ERCS/erc-7761.md

```diff
@@ -0,0 +1,121 @@
+---
+eip: 7761
```

Comment:
```suggestion
eip: 7774
```
Assigning next sequential EIP/ERC/RIP number.  7761 was already issued

Please also update the filename.